File: C#/SoftInvModel/Disponibilidad/DisponibilidadDTO.cs
using System;

namespace softinv.model
{
    public class DisponibilidadDTO
    {
        private int? disponibilidadId;
        private EmpleadoDTO empleado;
        private int? diaSemana;
        private DateTime? horaInicio;
        private DateTime? horaFin;
        public int? DisponibilidadId { get => disponibilidadId; set => disponibilidadId = value; }
        public EmpleadoDTO Empleado { get => empleado; set => empleado = value; }
        public int? DiaSemana { get => diaSemana; set => diaSemana = value; }
        public DateTime? HoraInicio { get => horaInicio; set => horaInicio = value; }
        public DateTime? HoraFin { get => horaFin; set => horaFin = value; }
    }
}

File: backend JAVA/SoftInvModel/src/main/java/pe/edu/pucp/softinv/model/Disponibilidad/DisponibilidadDTO.java
package pe.edu.pucp.softinv.model.Disponibilidad;
import pe.edu.pucp.softinv.model.Personas.EmpleadoDTO;

import java.sql.Time;

public class DisponibilidadDTO {
    private Integer disponibilidadId;
    private EmpleadoDTO empleado;
    private Integer diaSemana;
    private Time horaInicio;
    private Time horaFin;

    public Time getHoraFin() {
        return horaFin;
    }

    public void setHoraFin(Time horaFin) {
        this.horaFin = horaFin;
    }

    public Time getHoraInicio() {
        return horaInicio;
    }

    public void setHoraInicio(Time horaInicio) {
        this.horaInicio = horaInicio;
    }

    public Integer getDiaSemana() {
        return diaSemana;
    }

    public void setDiaSemana(Integer diaSemana) {
        this.diaSemana = diaSemana;
    }

    public EmpleadoDTO getEmpleado() {
        return empleado;
    }

    public void setEmpleado(EmpleadoDTO empleado) {
        this.empleado = empleado;
    }

    public Integer getDisponibilidadId() {
        return disponibilidadId;
    }

    public void setDisponibilidadId(Integer disponibilidadId) {
        this.disponibilidadId = disponibilidadId;
    }


}
